src/app/services/noisereduction/noisereductionmethods/rnnoise.py

```diff
@@ -0,0 +1,39 @@
+# from github: https://gist.github.com/kazuki/7c7b50ba9df082bfe9f03819201648eb
+
+from ctypes import byref, c_float, c_void_p, CDLL
+
+
+class RNNoise(object):
+
+    def __init__(self):
+        self._frameSize = 480
+        self._native = CDLL('/usr/local/lib/librnnoise.so')
```

Comment:
We should verify if this path exists, and raise a nice exception message if not.